raster-foundry/raster-foundry-api-spec
# Publish spec YAML file to version prefixed directory

## Overview

Publishes all YAML specifications to a version prefixed directory that looks like:

- `/spec/1.13.0/spec.yml`
- `/spec/1.13.1/spec.yml`
- `/spec/spec.yml` (points to same object as `/spec/1.13.1/spec.yml`)

### Checklist

- [x] Both spec files are updated

## Testing Instructions

See the Travis CI [build log](https://travis-ci.com/raster-foundry/raster-foundry-api-spec/builds/89361323) for the last commit.

Also, inspect the staging spec bucket using the AWS CLI:

```bash
$ aws s3 ls --human-readable s3://rasterfoundry-staging-docs-site-us-east-1/spec/
                           PRE 1.13.1/
2018-10-24 15:45:40   11.5 KiB prediction-spec.yml
2018-10-24 15:45:39  170.4 KiB spec.yml
$ aws s3 ls --human-readable s3://rasterfoundry-staging-docs-site-us-east-1/spec/1.13.1/
2018-10-24 15:45:38   11.5 KiB prediction-spec.yml
2018-10-24 15:45:39  170.4 KiB spec.yml
```

Fixes https://github.com/raster-foundry/raster-foundry-api-spec/issues/57
